pkg/model/dev.go

```diff
@@ -235,6 +235,10 @@ func (dev *Dev) validate() error {
 		return errBadName
 	}
 
+	if dev.SubPath != "" {
```

Comment:
can we remove it from the dev struct in this case? Or do we still need it under `services `? Also, could you update the manifest reference?

Reply:
Yes, it is still valid for `services`. I will update docs before the release is done